geode-core/src/main/java/org/apache/geode/internal/cache/TombstoneService.java

```diff
@@ -163,6 +212,10 @@ public void scheduleTombstone(LocalRegion r, RegionEntry entry, VersionTag destr
 
 
   private TombstoneSweeper getSweeper(LocalRegion r) {
+    TombstoneSweeper sweeper = this.regionTombstoneSweepers.get(r);
+    if (sweeper != null) {
```

Comment:
If every region has it's own sweeper, should it create one if one is somehow missing, or is this a signal that something bad has occurred that this region doesn't have as sweeper?